types/newman/index.d.ts

```diff
@@ -173,10 +173,14 @@ export interface NewmanRunExecutionAssertionError {
     test: string;
     message: string;
     stack: string;
-}export interface NewmanRunFailure {
+}
+export interface NewmanRunFailure {
     error: NewmanRunExecutionAssertionError;
     /** The event where the failure occurred */
     at: string;
+    source: any;
+    parent: any;
+    cursor: any;
```

Comment:
A guess based on https://github.com/postmanlabs/newman/blob/d984e5d042f131bc43b4f359044f8b772e5b803d/test/unit/run-summary.test.js#L73
and other usage
https://github.com/postmanlabs/newman/blob/e08983dad7798531a4f42e3747b3a6d101ac949e/lib/run/summary.js#L238-L241
```suggestion
    cursor: { ref: string } | {};
```